WordPress/openverse
# Enforce braces for `if`-`else` and `switch`-`case`

## Fixes
Fixes #3386 by @dhruvkb

This PR incorporates changes to the ESLint configuration to enforce the use of braces in one-line if-else statements and case blocks inside switch statements. This update aims to enhance code readability and prevent potential issues in the future. The specific ESLint rules added are:

- Enabled ESLint's built-in curly rule.
- Enabled the unicorn plugin's switch-case-braces rule.

### Testing Instructions
1. Run ESLint on the project to check for adherence to the new rules: `pnpm run eslint`
2. If necessary, automatically fix fixable issues using: `pnpm run eslint --fix`

### Checklist


- [x] I ran the project locally and verified that there are no visible errors.
